src/Text/LLVM/AST.hs

```diff
@@ -1647,10 +1647,36 @@ data DISubprogram' lab = DISubprogram
 
 type DISubprogram = DISubprogram' BlockLabel
 
-data DISubrange = DISubrange
-  { disrCount      :: Int64
-  , disrLowerBound :: Int64
-  } deriving (Data, Eq, Generic, Ord, Show, Typeable)
+-- | The DISubrange is a Value subrange specification, usually associated with
+-- arrays or enumerations.
+--
+-- * Early LLVM: only 'disrCount' and 'disrLowerBound' were present, where both
+--   were a direct signed 64-bit value.  This corresponds to "format 0" in the
+--   bitcode encoding.
```

Comment:
I don't think "format 0" will mean anything to most readers unless you cite where this is used in the LLVM source code—worth doing so here.

Reply:
Cited in the location of your comment below, but added a reference here.